bot-components/GitHub_mutations.ml

```diff
@@ -154,7 +154,7 @@ let create_check_run ~bot_info ?conclusion ~name ~repo_id ~head_sha ~status
         | {createCheckRun= Some {checkRun= Some {url}}} ->
             Ok url
         | _ ->
-            Error (f "Error while creating check run.") )
+            Error (f "Warning: Check run could not be created.") )
```

Comment:
Actually, as we have determined together, since we are inside a bind, it means that the mutation has still returned successfully, so what we should warn about is not that the check run could not be created but that we didn't get the URL back.